### DISCUSSION TITLE
How can I know if the patient has pre-existing atrial fibrillation?

### DISCUSSION THREAD

**Discussion by laithomari** [MIT-LCP/mimic-code#1514] [MIMIC-IV]
Hello. I am trying to study the incidence of atrial fibrillation in critical care setting. My question is how can I know if the patient already had atrial fibrillation before admission to the hospital? I tried checking the diagnosis_icd table, however, in ICD-9 codes there are no specific code for chronic atrial fibrillation. Another but related question would be is that how can I identify new-onset atrial fibrillation? Thanks

**Comment by alistairewj** [comment]
You could look at the approach we took here, which specifically looks at new-onset atrial fibrillation in MIMIC: https://www.sciencedirect.com/science/article/pii/S0883944121002525

AF before ICU admission was defined as:

- a mention of atrial fibrillation in the past medical history section from the discharge summary
- a mention of common AF medications in the medications on admission section of the discharge summary
    - specifically digoxin, lanoxin, amiodarone, cordarone, nextrone, pacerone, sotalol, betapace, sorine, sotylize, flecainide, and tambocor 
- a rhythm of atrial fibrillation charted in MetaVision within the first 3 hours of ICU admission

There is now a code for chronic atrial fibrillation (ICD-10 I48.2). It would make sense to use that as well (our study pre-dated the use of ICD-10 in MIMIC).
